packages/editor/src/components/provider/index.native.js

```diff
@@ -256,7 +255,9 @@ class NativeEditorProvider extends Component {
 
 	componentDidUpdate( prevProps ) {
 		if ( ! prevProps.isReady && this.props.isReady ) {
-			const blocks = this.props.blocks;
+			const blocks =
+				this.props.getEditedPostAttribute( 'blocks' ) ||
+				parse( this.props.getEditedPostContent() );
```

Comment:
There's something terribly wrong here on the native side, but that's out of the scope of this PR. The blocks in the block editor store should be available when the editor is ready, yet they aren't (causing some test failures). I've just restore the logic from `getEditorBlocks`, though obviously it's going to parse the content here.